src/java.desktop/share/classes/javax/swing/DebugGraphics.java

```diff
@@ -294,67 +320,87 @@ public void setXORMode(Color aColor) {
         if (debugLog()) {
             info().log(toShortString() + " Setting XOR mode: " + aColor);
         }
-        graphics.setXORMode(aColor);
+        if (graphics != null) {
+            graphics.setXORMode(aColor);
+        }
     }
 
     /**
      * Overrides <code>Graphics.getClipBounds</code>.
      */
     public Rectangle getClipBounds() {
+        if (graphics == null) {
+            return null;
+        }
         return graphics.getClipBounds();
     }
 
     /**
      * Overrides <code>Graphics.clipRect</code>.
      */
     public void clipRect(int x, int y, int width, int height) {
+        if (graphics == null) {
+            return;
+        }
         graphics.clipRect(x, y, width, height);
         if (debugLog()) {
             info().log(toShortString() +
-                " Setting clipRect: " + (new Rectangle(x, y, width, height)) +
-                " New clipRect: " + graphics.getClip());
+                    " Setting clipRect: " + (new Rectangle(x, y, width, height)) +
+                    " New clipRect: " + graphics.getClip());
```

Comment:
Why this line change is needed? Probably you have added some tabs..Remove this change and similarly from other logs down below.

Reply:
I think auto indentation has caused this, will undo these tabs changes.